src/app/launcher/create-app/project-progress-createapp-nextstep/project-progress-createapp-nextstep.component.ts

```diff
@@ -26,9 +26,16 @@ export class ProjectProgressCreateappNextstepComponent implements OnInit, OnDest
   }
 
   ngOnInit() {
-    this.subscriptions.push(this.projectProgressService.getProgress().subscribe((progress) => {
-      this._progress = progress;
-    }));
+    this.subscriptions.push(this.projectProgressService
+      .getProgress(this.launcherComponent.summary.uuidLink)
+      .subscribe((progress) => {
+        this._progress = progress;
+      }));
+          this.projectProgressService.connect(this.launcherComponent.summary.uuidLink);
```

Comment:
Please do not store the uuidLink in the summary. Can we have the progress component should fetch the uuidLink itself instead having the summary component store it?

Reply:
sure. i will handle it.